[2023-09-04T06:02:13.833+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [queued]>
[2023-09-04T06:02:13.852+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [queued]>
[2023-09-04T06:02:13.852+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:13.877+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-15 00:00:00+00:00
[2023-09-04T06:02:13.888+0000] {standard_task_runner.py:57} INFO - Started process 5207 to run task
[2023-09-04T06:02:13.896+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-15T00:00:00+00:00', '--job-id', '386', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp56ub4578']
[2023-09-04T06:02:13.899+0000] {standard_task_runner.py:85} INFO - Job 386: Subtask delete_data_from_table
[2023-09-04T06:02:13.970+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:14.099+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-15T00:00:00+00:00'
[2023-09-04T06:02:14.100+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-15' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:02:14.113+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.125+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:14.139+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-15' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:02:14.145+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:02:14.162+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220315T000000, start_date=20230904T060213, end_date=20230904T060214
[2023-09-04T06:02:14.225+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:14.250+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:58.665+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [queued]>
[2023-09-04T06:04:58.716+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [queued]>
[2023-09-04T06:04:58.717+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:58.775+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-15 00:00:00+00:00
[2023-09-04T06:04:58.788+0000] {standard_task_runner.py:57} INFO - Started process 6525 to run task
[2023-09-04T06:04:58.808+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-15T00:00:00+00:00', '--job-id', '932', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp8adguti2']
[2023-09-04T06:04:58.814+0000] {standard_task_runner.py:85} INFO - Job 932: Subtask delete_data_from_table
[2023-09-04T06:04:58.998+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-15T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:59.314+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-15T00:00:00+00:00'
[2023-09-04T06:04:59.316+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-15' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:59.348+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:59.370+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:59.393+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-15' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:59.402+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:59.434+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220315T000000, start_date=20230904T060458, end_date=20230904T060459
[2023-09-04T06:04:59.502+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:59.831+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
